The closest living relatives to humans on the evolutionary tree are chimpanzees. But how long ago did our most recent common ancestor live? This question motivated my research in theoretical evolutionary genetics; I have worked on this project for the last year with Professor Matveyev since I finished my undergraduate studies. During this time, I have cemented my desire to go to graduate school and learned which flavor of problems I prefer. I want to investigate systems in which collective behavior leads to emergent phenomena, like pattern formation. The goal is to identify the minimal set of rules needed to explain an observed phenomenon and combine them into a mathematical theory that can generalize to more complicated scenarios.

In my research, our goal was to estimate a quantity called the coalescent time. It is defined as follows: given a nucleotide in the DNA of any two organisms, how many generations ago did they share the ancestor from whom they inherited that nucleotide. Current (paleontological) methods put the coalescent time for humans and chimpanzees to be between 4 and 13 million years ago, which is quite a wide range. We hope that our model will validate and make more precise this estimate as well as be applied to organisms for which we do not have well-preserved records. With our method, under some simplifying assumptions, we can compute probability distributions for the coalescent time of a subset of nucleotides of a pair of DNAs. We also check the statistical soundness of these distributions by simulating DNA evolution trees. Currently, we are expanding our method to model an additional factor: selection.

For our model, we assume that there are two types of nucleotides (instead of the 4 in reality), individuals reproduce sexually, generations are distinct, of equal size and don’t interbreed, and parent-child relations are randomly established. We treat coalescent time as a random variable to account for mutations and recombination, which add randomness to the inheritance process. Under these assumptions, the inheritance tree can be represented by a random directed graph. The nodes are individuals. Each node has two directed edges coming in from two randomly chosen parent nodes from the previous generation. Each node is also labeled by its generation and the DNA of the individual. The DNA of an individual is generated from the DNA of its parents by recombination, a Poisson process that alternates between copying sections of the parents’ DNA. For each nucleotide in the DNA, we also track which parent it came from. To find the coalescent time of a nucleotide for two individuals in a generation, we follow the inheritance line from each individual through the parents until the two lines merge. The number of generations ago that they merged is the coalescent time. Since these graphs are generated stochastically there is a probability distribution on them, which induces a probability distribution on the coalescent time.

However, the probability distribution on the graphs is intractable. To address this challenge, we choose one end of the DNA as the beginning and assume an exponential distribution for the coalescent time of that nucleotide. Then, we derive a differential equation that relates the marginal distributions for the coalescent times of adjacent nucleotides. These are related because recombination copies over segments of the parents' DNA at a time. This differential equation allows us to iteratively calculate, in integral form, the probability distribution of coalescent times for the next nucleotide in the DNA. For the first few nucleotides in the sequence, we can approximate the distributions numerically. 

We are now extending this method to include selection, which models the competitive fitness of the variants of a nucleotide. This complicates the story because, to simulate the inheritance of a nucleotide to a child, we need to keep track of the genetic composition of the entire parent generation. So far, we have simplified this picture in the large population limit, where the probability of inheriting a nucleotide can be approximated using the stationary phase method, a tool I learned in Quantum Field Theory. Our next goal is to use these simulated trees to complete modeling the dependence of coalescent time on selection.

Throughout this, I have really enjoyed incrementally building up our theory. We started by understanding the basic mathematical axioms that govern the simplest case of a system and then gradually generalized our model by introducing one new parameter at a time. It was very rewarding to see, through this process, how each parameter contributes to the complexity of the phenomena we studied. I’ve also enjoyed combining analytics with simulation and leveraging tools I learned from physics classes, such as the stationary phase method, alongside tools from statistics, math, and computer science. I would like to continue such interdisciplinary work in graduate school. This makes Columbia Biological Sciences a very appealing program. Another important quality is the ability to work closely with state of the art experiments happening at Columbia. My research required many simplifications to make our idealized model for genetic dynamics tractable. By taking experiments/data into consideration, I will be able to better understand the implications and validity of these assumptions and make my research more tangible. Since science is driven by verifying hypotheses and models through experiment, bridging theory and real data is valuable to me. At Columbia, I would like to work with Professor Peter Andolfatto because of his work on speciation and hybridization. These two processes are something I completely overlooked in my own research, but vital in realistic genetic inheritance processes and evolution of species. I have been looking into extending my current model to include these factor. I am also interested in working with Professor Harmen Bussemaker on studying gene regulatory networks. Having studied DNA mutations, I am interested in how these mutations affect gene expression regulation. This type of research appeals to me because it is highly analytical and computational in nature, and also can be very impactful in the medical world because mutations lead to changes in gene expression which can lead to debilitating diseases like cancer. Columbia Biological Sciences provides many fantastic research opportunities, and the potential to bring in new theoretical and computational ideas inspired by the cutting edge research happening all over campus; I would love to mature as a scientist in this environment.
