Broadly speaking, I want to investigate physical and biological systems with interesting observables--population dynamics, pattern formation or emergent phenomena from collective behavior--that are driven by fundamental physical principles. The goal is to identify the underlying rules governing these systems and construct a mathematical theory that can describe the phenomena and generalize to more complicated scenarios. My interest in this flavor of problems was cemented during my work in theoretical evolutionary genetics with Professor Matveyev.

  In my research, our goal was to estimate, given a nucleotide in the DNA of any two organisms, how many generations ago did they share the ancestor from whom they inherited that nucleotide. For example, our closest relatives on the evolutionary tree are chimpanzees; how long ago did our most recent common ancestor live? I am building a probabilistic model that can estimate this quantity, called the coalescent time. Current (paleontological) methods put the coalescent time for humans and chimpanzees to be somewhere between 4 and 13 million years ago, which is quite a wide range. We hope that our model will validate and make more precise this estimate as well as be applied to organisms for which we do not have well-preserved records. So far, under certain simplifying assumptions, we have computed probability distributions for the coalescent time of a subset of nucleotides of a given pair of DNAs. We also check the statistical soundness of these distributions by simulating DNA evolution trees. Currently, we are expanding our methods to model an additional factor: selection.

We make the following modeling assumptions: there is sexual reproduction, generations are distinct and don’t interbreed, all generations are of equal size, parent-child relations are randomly established, and there are two types of nucleotides (instead of the 4 in reality). We treat coalescent time as a random variable to account for mutations and recombination, which add randomness to the inheritance process. Under these assumptions, the inheritance tree can be represented by a random directed graph. The nodes are individuals. Each node has two directed edges coming in from two randomly chosen nodes (parents) from the previous generation. Each node is also labeled by which generation it belongs to, and the DNA of the individual. The DNA at a node is generated from the DNA of its parents by recombination, which is a Poisson process that alternates between copying sections of the parents’ DNA. For each nucleotide, we track which parent it came from. To find the coalescent time of a nucleotide for two individuals in a generation, we follow the inheritance line from each individual through the parents up until the two lines merge. The number of generations ago that they merged is the coalescent time. Since these graphs are generated stochastically there is a probability distribution on them, which induces a probability distribution on the coalescent time.

However, the probability distribution on the graphs is intractable. To address this challenge, we choose one end of the DNA as the beginning and assume that the distribution of coalescent time for that nucleotide is exponential. Then, we derive a differential equation that relates the marginal distributions for the coalescent time of adjacent nucleotides. This allows us to iteratively calculate, in integral form, the probability distribution of coalescent times for the next nucleotide in the DNA. For the first few nucleotides in the sequence, we can approximate the distributions numerically. We are now extending this method to include selection, which models the competitive fitness of the variants of a nucleotide. This complicates the story because, to simulate the inheritance of a nucleotide to a child, we need to keep track of the genetic composition of the entire parent population. So far, we have simplified this picture in the large population limit and studied it with the stationary phase method, a tool I learned in Quantum Field Theory.

Throughout this, I have enjoyed incrementally building up our theory by leveraging both analytical and computational tools from physics, statistics, mathematics, and computer science. We started by understanding the basic mathematical axioms that govern the simplest case of a system and then gradually introduced one new parameter at a time, thus generalizing our model. It was very rewarding to see, through this process, how each parameter contributed to the complexity of the phenomena we studied. However, to construct a careful, bottom-up model, we neglected many details--the different ways DNA can mutate, how DNA can affect reproduction patterns, how population structure and dispersal affects evolutionary genetic dynamics--which complicate the study of evolutionary genetics. For my graduate studies, I would like to delve deeper into this field. This motivates my interest in the Quantitative and Computational Biology (QCB) program at Princeton where there are a lot of people working on similar problems. In particular, I would like to work with Professor Bridgett vonHoldt or Professor Joshua Akey on evolutionary genetics because of the close connection to data in their work. Going forward, I want to make my research more tangible by incorporating real data and working with experimentalists. Since I am not restricting myself to working just on evolutionary genetics, I am also very interested in Professor John Storey’s theoretical and statistical model work on high-dimensional data analysis problems in genomics. It would be rewarding to work on genetic statistics because it is such an indispensable tool in analyzing and understanding DNA. QCB provides many research and study opportunities that I would love to take advantage of to continue my studies of genetic evolution. It would be very exciting to bring into my research new theoretical and computational ideas inspired by the cutting edge research happening all over Princeton. 
